productivity-suite/app/fragments/news/src/components/story.tsx

```diff
@@ -0,0 +1,59 @@
+import { Component, Show } from "solid-js";
+import { A } from "solid-start";
+
+import type { IStory } from "../types";
+
+const Story: Component<{ story: IStory }> = (props) => {
+  const hnUrl = (path) => `https://news.ycombinator.com${path}`;
+
+  return (
+    <li class="news-item">
+      <span class="score">{props.story.points}</span>
+      <span class="title">
+        <Show
+          when={props.story.url}
+          fallback={
+            <A href={hnUrl(`/item?id=${props.story.id}`)}>
+              {props.story.title}
+            </A>
+          }
+        >
+          <a
+            href={hnUrl(`/item?id=${props.story.id}`)}
+            target="_blank"
+            rel="noreferrer"
+          >
+            {props.story.title}
+          </a>
+          <span class="host"> ({props.story.domain})</span>
+        </Show>
+      </span>
+      <br />
+      <span class="meta">
+        <Show
+          when={props.story.type !== "job"}
+          fallback={
+            <A href={hnUrl(`/item?id=${props.story.id}`)}>
```

Comment:
why these are `A` instead of `a` (like above)? do we still have some client side navigation? if not I guess we could also add the `target="_blank"`?